liquidhaskell-boot/src/Language/Haskell/Liquid/Bare/Elaborate.hs

```diff
@@ -566,7 +566,7 @@ renameBinderSort f = rename
 
 
 mkHsTyConApp ::  IdP GhcPs -> [LHsType GhcPs] -> LHsType GhcPs
-mkHsTyConApp tyconId tyargs = nlHsTyConApp NotPromoted Prefix tyconId (map (HsValArg noExtField) tyargs)
+mkHsTyConApp tyconId tyargs = nlHsTyConApp NotPromoted Prefix tyconId (map HsValArg tyargs)
```

Comment:
This diff would not be necessary if this definition was moved to `Liquid.GHC.API.Extra`.

Reply:
This one I agree with, since this makes sense to move to `API.Extra`.